proposals.adoc

```diff
@@ -0,0 +1,112 @@
+
+
+== Infrastructure
+
+=== GitHub
+
+Proposals are managed as GitHub issues in the https://github.com/bisq-network/proposals/issues[bisq-network/proposals] repository.
+
+TIP: If you're an active Bisq contributor, consider https://help.github.com/articles/watching-and-unwatching-repositories/[watching] this repository to be notified of new proposals and discussions around them. You can always https://help.github.com/articles/subscribing-to-and-unsubscribing-from-notifications/[unsubscribe] from specific issues you're not interested in.
+
+=== Slack
+
+The `#proposals` Slack channel is available for discussion and notifications about proposal issue activity.
+
+
+== Roles and responsibilities
+
+=== Submitter
+
+The contributor(s) who write a proposal and carry it through to completion. **Submitters are 100% responsible for the success of their proposals.**
+
+=== Reviewer
+
+Other contributors who read, discuss and react to proposals. **Any contributor _may_ review a proposal, but no contributor is obligated to do so.** This intentionally puts the onus on the submitter to ensure their proposal is relevant and well-written per the guidelines below.
+
+=== Maintainer
+
+The contributor(s) who have administrative rights over the `bisq-network/proposals` GitHub repository, monitor the `#proposals` Slack channel, enforce the process detailed below, and write https://github.com/bisq-network/roles/issues/30[monthly reports] on proposals activity. **Maintainers have no special authority to approve or reject proposals.**
+
+
+== Process
+
+=== Step 0. Research
+
+Before you submit a proposal, **do your homework!**
+
+ . Discuss your idea with other contributors to see if a proposal is worth submitting at all;
+ . Search through existing proposals (both https://github.com/bisq-network/proposals/issues?utf8=%E2%9C%93&q=is%3Aissue+[open and closed]) to see whether something similar has already been proposed;
+ . Notice which among those proposals have been accepted and rejected, and why;
+ . Read this document to fully understand the proposal process and guidelines.
+
+=== Step 1. Submit
+
+Create a new GitHub issue in the `bisq-network/proposals` repository containing the text of your proposal, written and formatted per the guidelines below.
+
+A maintainer will quickly review your proposal and will either (a) assign it to you to indicate your ownership and responsibility over it, or (b) close it and label it as `was:incorrect` if it does not follow the guidelines below.
+
+=== Step 2. Review
+
+Once a proposal is submitted, a two-week review period follows. During this period, interested reviewers should read, discuss and ultimately react to the proposal as follows:
+
+ - 👍: I agree with the proposal and want to see it enacted
```

Comment:
This renders as square. I think something else was intended here.

Reply:
Interesting, looking into it.

Reply:
This is what I see in the deploy preview at https://deploy-preview-40--bisq-docs.netlify.com/proposals.html:

![image](https://user-images.githubusercontent.com/301810/39127989-e65f32f0-4706-11e8-97a2-0c433631aa01.png)

Can you share a screenshot of what you see?

Reply:
![proposals](https://user-images.githubusercontent.com/431064/39128107-354f1678-4707-11e8-8dd3-4937fa897c13.png)

Reply:
![screen shot 2018-04-23 at 15 03 28](https://user-images.githubusercontent.com/1465310/39128242-96b34f7e-4707-11e8-9f44-5228bf3aa246.png)

Reply:
I guess I miss some font.